invenio_rdm_records/resources/serializers/csl/schema.py

```diff
@@ -108,7 +109,11 @@ def get_language(self, obj):
 
     def get_doi(self, obj):
         """Get DOI."""
-        return obj["pids"].get("doi", {}).get("identifier", missing)
```

Comment:
without this tweak, the exported CSL would include the `https://doi.org/` prefix twice, e.g.:
`Miksa, T. (2023). test record for base. My Site. https://doi.org/https://doi.org/10.1234/abcq`